enterprise/utils.py

```diff
@@ -546,11 +547,15 @@ def serialize_notification_content(
     email_items = []
     for user in users:
         login_or_register = 'register' if is_pending_user(user) else 'login'
-        destination_url = '{site}/{login_or_register}?next={course_path}'.format(
-            site=lms_root_url,
-            login_or_register=login_or_register,
-            course_path=course_path
-        )
+        # if we have an activation link for a license, use that rather than the course URL
+        if activation_links is not None and activation_links.get(user.email) is not None:
```

Comment:
I think `if activation_links is not None` can be shorted to `if activation_links`